SomeFile.java

```diff
@@ -0,0 +1,19 @@
+package com.test.common;
+
+public class SomeFile {
+
+  private static String message = "some text";
+
+  public void SomeMethod() {
+    otherMethod(3);
+    anotherMethod(true, false);
```

Comment:
Use parameter comments to document ambiguous literals.

[Show suggested change](http://google.com)

Reply:
anotherMethod(true, /* maybeNot= */ false);